inline.go

```diff
@@ -75,6 +76,9 @@ func (p *parser) inline(data []byte) {
 			if p.inlineCallback[data[end]] != nil {
 				if end+1 < len(data) && data[end] == 'h' && data[end+1] != 't' {
 					end++
+				} else if strings.ContainsRune("hHmMfF", rune(data[end])) && !isAutoLink(p, data, end) {
```

Comment:
I would agree this is more readable if not for the cast of byte to rune, which spoils all the gain, if there was any.

But the style is a secondary question to me here, performance is the primary. This is one of the hottest paths in the library and we're calling a (potentially) more expensive call for nearly every byte of input. Of course, `ContainsRune` is implemented in terms of `IndexByte`, so there should not be much difference asymptotically, but the little constant overhead might be very noticeable.